srcpkgs/armadillo/template

```diff
@@ -20,7 +20,7 @@ if [ -z "$CROSS_BUILD" ]; then
 fi
 
 armadillo-devel_package() {
-	depends="${makedepends}"
+	depends="${makedepends} armadillo>=${version}_${revision}"
```

Comment:
please use `${sourcepkg}>=${version}_${revision}"`, see https://github.com/void-linux/void-packages/blob/master/Manual.md#subpackages

Reply:
I wasn't quite sure what the variable was called.
I didn't have the patience to read the entire manual, so I searched for `basepkg` with no success.